packages/agoric-cli/src/commands/vaults.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable @jessie.js/no-nested-await */
+// @ts-check
+/* eslint-disable func-names */
+/* global fetch */
+import { Command } from 'commander';
+import { makeRpcUtils } from '../lib/rpc.js';
+import { makeOpenSpendAction } from '../lib/vaults.js';
+import { outputAction } from '../lib/wallet.js';
+
+/** @typedef {import('@agoric/smart-wallet/src/offers').OfferSpec} OfferSpec */
+/** @typedef {import('@agoric/smart-wallet/src/offers').OfferStatus} OfferStatus */
+/** @typedef {import('@agoric/smart-wallet/src/smartWallet').BridgeAction} BridgeAction */
+
+const {
+  vstorage,
+  fromBoard: _fromBoard,
+  agoricNames,
+} = await makeRpcUtils({ fetch });
```

Comment:
ambient access to the network. sigh.

Reply:
`makeRpcUtils` takes `fetch` to make that an ocap.

Think of each command file as a script start. It's gathering the authorities. This once comes from global.